Packs/VMRay/Integrations/VMRay/VMRay_test.py

```diff
@@ -53,20 +53,19 @@ def test_upload_sample_command(mocker):
     assert mocker_output.call_args.args[0] == expected_output
 
 
-def test_encoding_file_name():
-    """
-    Given:
-        A string representing a file name with backslashes
-    When:
-        encode_file_name is running
-    Then:
-        Verify the output of encode_file_name is the same as the input string (in bytes) without it's backslashes
-    """
-    file_name = '\\test\\encode\\file\\name'
-    expected_output = 'testencodefilename'
+@pytest.mark.parametrize(
+    "file_name, expected",
+    [
+        ("abc.exe", b"abc.exe"),
+        ("<>:\"/\\|?*a.exe", b"a.exe"),
+        ("\\test\\encode\\file\\name", b"testencodefilename"),
+        ("ñá@.exe", b"\xc3\xb1\xc3\xa1@.exe"),
+    ]
+)
+def test_encoding_file_name(file_name, expected):
```

Comment:
Please add a doc string with Given When, Then, with an updated explanation about the test.